pkg/porter/dependencies.go

```diff
@@ -181,6 +181,11 @@ func (e *dependencyExecutioner) prepareDependency(dep *queuedDependency) error {
 	dep.CNABFile = cachedDep.BundlePath
 	dep.RelocationMapping = cachedDep.RelocationFilePath
 
+	invocationImage := cachedDep.Bundle.InvocationImages[0]
+	if invocationImage.Digest == "" {
+		return fmt.Errorf("no content digest is present for dependency image %s", invocationImage.Image)
```

Comment:
So if I saw this error, I would have no idea what to do about it or what the problem is. Let's try to explain the security issue and how the invocation image for the bundle doesn't have a digest so we can't verify that it is is the image referenced by the bundle and hasn't been tampered with. Or something like that.

Basically we need to throw the bundle under the bus, say it's invalid and could be compromised.

Reply:
🚌 💨

Reply:
Error text has been updated to better convey the issue around this scenario.  Thank you!